elastic/kibana
# [Fleet] Enforce unique names on download binary sources

## Summary
Fixes https://github.com/elastic/kibana/issues/136505

Enforce name uniqueness on download binary sources


### Tests -  Create

- Navigate to settings -> `Agent Binary Download` section
- Create a new download source with the same name of an existing one, save & deploy when prompted
- You should see an error

![Screenshot 2022-07-19 at 16 33 01](https://user-images.githubusercontent.com/16084106/179776545-0d01206c-b164-44f9-98bc-819ea3e7f1df.png)

### Tests -  Update
- Update one of the entries the same name of an existing one, save & deploy when prompted
- You should see an error


### Checklist

- [x ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
